libs/vkd3d/utils.c

```diff
@@ -483,6 +483,25 @@ const struct vkd3d_format *vkd3d_get_format(const struct d3d12_device *device,
     return format->dxgi_format ? format : NULL;
 }
 
+const struct vkd3d_format *vkd3d_format_footprint_for_plane(const struct d3d12_device *device,
```

Comment:
Not sure I like this design; I'd rather have some sort of
```
struct {
  uint32_t bytes_per_pixel;
  VkExtent2D block_size;
};
```
that we can query from a vkd3d_format for any given plane (maybe omitting `block_size` for now since we don't support subsampled formats yet, but we'll probably need to at some point).

Reply:
Fixed.